portal/static/portal/sass/partials/_buttons.scss

```diff
@@ -242,6 +242,15 @@ button,
   color: $color-text-primary;
 }
 
+.button--secondary--aimmo--download {
+  background: $color-button-aimmo-materials;
```

Comment:
I would give that var a different name

Reply:
This will be obsolete after my new commit